pkg/api/postage_test.go

```diff
@@ -232,6 +232,51 @@ func TestPostageGetStamps(t *testing.T) {
 	})
 }
 
+func TestPostageGetNonExistingStamps(t *testing.T) {
+	b := postagetesting.MustNewBatch()
+	b.Value = big.NewInt(20)
+	mp := mockpost.New()
+	ts, _, _, _ := newTestServer(t, testServerOptions{DebugAPI: true, Post: mp, BatchStore: nil, BlockTime: big.NewInt(2)})
+
+	t.Run("expired stamp", func(t *testing.T) {
+		jsonhttptest.Request(t, ts, http.MethodGet, "/stamps", http.StatusOK,
+			jsonhttptest.WithExpectedJSONResponse(&api.PostageStampsResponse{Stamps: []api.PostageStampResponse{}}),
+		)
+	})
+}
+
+func TestPostageGetAllStamps(t *testing.T) {
```

Comment:
You could have just 1 test which has 2 subtests. In one case you can pass the `all=true` param and other case will not pass it.

It would be better to have multiple stamps as well. Maybe 1 valid and 1 expired stamp. If you need to modify some of the mock classes for this, that should not be a problem. Mock classes are meant for tests, so if they dont cover certain things we can always add them.